### ISSUE TITLE
php 5.5. on branch 2.0 Unable to complete the requested operation.

### ISSUE THREAD

**Issue by radinosk** [spress/Openshift-spress-cartridge#1]
Shell command '/sbin/runuser -s /bin/sh 56f5829189f5cfb3c20000b8 -c "exec /usr/bin/runcon 'unconfined_u:system_r:openshift_t:s0:c6,c583' /bin/sh -c \"/var/lib/openshift/56f5829189f5cfb3c20000b8/spress/bin/install --version 5.5\""' returned an error. rc=1

Unable to open /etc/scl/prefixes/php55!

**Comment by yosymfony** [comment]
yes, branch 2.0 fail. Currently I'm busy with the release of Spress 2.1. To run Spress 2.x on Openshift we need another cartridge as a base such as https://github.com/boekkooi/openshift-cartridge-php
